webOSTVjs-1.1.0/webOSTV.js

```diff
@@ -0,0 +1 @@
y{i.open("GET",t||"appinfo.json",!0),i.send(null)}catch(e){n({status:404})}}else e&&e(o)},t.fetchAppRootPath=function(){var e=window.location.href;if("baseURI"in window.document)e=window.document.baseURI;else{var t=window.document.getElementsByTagName("base");t.length>0&&(e=t[0].href)}var n=e.match(new RegExp(".*://[^#]*/"));return n?n[0]:""},t.platformBack=function(){if(window.PalmSystem&&window.PalmSystem.platformBack)return window.PalmSystem.platformBack()}},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(1),o=n(0),i=function(e){return e&&e.__esModule?e:{default:e}}(o),s={},a=function(e){if(0===Object.keys(s).length){try{var t=JSON.parse(window.PalmSystem.deviceInfo);s.modelName=t.modelName,s.version=t.platformVersion,s.versionMajor=t.platformVersionMajor,s.versionMinor=t.platformVersionMinor,s.versionDot=t.platformVersionDot,s.sdkVersion=t.platformVersion,s.screenWidth=t.screenWidth,s.screenHeight=t.screenHeight}catch(e){s.modelName="webOS Device"}s.screenHeight=s.screenHeight||window.screen.height,s.screenWidth=s.screenWidth||window.screen.width,i.default.tv&&(new r.LS2Request).send({service:"luna://com.webos.service.tv.systemproperty",method:"getSystemInfo",parameters:{keys:["firmwareVersion","modelName","sdkVersion","UHD"]},onSuccess:function(t){if(s.modelName=t.modelName||s.modelName,s.sdkVersion=t.sdkVersion||s.sdkVersion,s.uhd="true"===t.UHD,t.firmwareVersion&&"0.0.0"!==t.firmwareVersion||(t.firmwareVersion=t.sdkVersion),t.firmwareVersion){s.version=t.firmwareVersion;for(var n=s.version.split("."),r=["versionMajor","versionMinor","versionDot"],o=0;o<r.length;o+=1)try{s[r[o]]=parseInt(n[o],10)}catch(e){s[r[o]]=n[o]}}e(s)},onFailure:function(){e(s)}})}else e(s)};t.default=a},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r={isShowing:function(){return!!PalmSystem&&(PalmSystem.isKeyboardVisible||!1)}};t.default=r},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=function(){var e={};if(window.PalmSystem){if(window.PalmSystem.country){var t=JSON.parse(window.PalmSystem.country);e.country=t.country,e.smartServiceCountry=t.smartServiceCountry}window.PalmSystem.timeZone&&(e.timezone=window.PalmSystem.timeZone)}return e};t.default=r},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});t.default="1.1.0"},function(e,t,n){"use strict";function r(e){return e&&e.__esModule?e:{default:e}}Object.defineProperty(t,"__esModule",{value:!0}),t.systemInfo=t.service=t.platform=t.platformBack=t.libVersion=t.keyboard=t.fetchAppRootPath=t.fetchAppInfo=t.fetchAppId=t.deviceInfo=void 0;var o=n(2),i=n(1),s=r(i),a=n(3),c=r(a),u=n(4),l=r(u),f=n(0),d=r(f),m=n(5),p=r(m),v=n(6),h=r(v);t.deviceInfo=c.default,t.fetchAppId=o.fetchAppId,t.fetchAppInfo=o.fetchAppInfo,t.fetchAppRootPath=o.fetchAppRootPath,t.keyboard=l.default,t.libVersion=h.default,t.platformBack=o.platformBack,t.platform=d.default,t.service=s.default,t.systemInfo=p.default}]);
```

Comment:
do we need both files here? `webOSTV-dev.js` and `webOSTV.js`?

Reply:
Those are provided by LG and I updated the whole bundle.